public/dist/assets/styles/core.css

```diff
@@ -489,13 +489,21 @@ blockquote {
   display: flex;
   flex-direction: column;
   justify-content: flex-start;
+  padding: 0 2.5rem;
+  quotes: auto;
 }
 blockquote > * + * {
   margin-top: 1rem;
 }
 blockquote > * {
   font-weight: bold;
 }
+blockquote p:first-child:before {
```

Comment:
`blockquote` could have [other content that paragraphs](https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element) so it might be better to add the pseudo classes on the `blockquote` itself. However, the flex container will display the quotes on different lines so there might be more rules to add.

Reply:
Thanks @NicolaSaunders, this does resolve the issue